presto-native-execution/presto_cpp/main/tests/ServerOperationTest.cpp

```diff
@@ -29,7 +29,7 @@ namespace facebook::presto {
 class ServerOperationTest : public testing::Test {
   void SetUp() override {
     FLAGS_velox_memory_leak_check_enabled = true;
-    memory::MemoryManager::testingSetInstance({});
+    exec::test::OperatorTestBase::SetUp();
```

Comment:
Should we also call exec::test::OperatorTestBase::TearDown() to pair up semantically?